extensions/python/PYTHON.md

```diff
@@ -102,10 +102,12 @@ NON_BLANK = 5
 PORT = 6
 ```
 
+The last parameter of addProperty is the controller service type. If the property is a controller service, the controller service type should be provided. It should be the non-qualified type name of the controller service. Currently SSLContextService is the only controller service type supported.
+
 ```python
 def onInitialize(processor):
   processor.setSupportsDynamicProperties()
-  processor.addProperty("property name","description","default value", True /*required*/, False /*expression language supported*/, False /*sensitive*/, 1 /*property type code*/)
+  processor.addProperty("property name", "description", "default value", True /*required*/, False /*expression language supported*/, False /*sensitive*/, 1 /*property type code*/, None /*controller service type name*/)
```

Comment:
These `/**/` style comments are not valid python. I'm assuming this comes from the NiFi Python API. Otherwise I'd consider changing to keyword arguments.

Reply:
Good point, I added comment for the arguments in 0b35f1cf419e8b57955f65f9192cf39b8d8a5fde. Due to the implementation of `PyProcessor::addProperty(PyProcessor* self, PyObject* args)`  these are handled as positional arguments, so using keyword arguments would be misleading.